src/modules/mount/main.py

```diff
@@ -61,6 +61,97 @@ def get_btrfs_subvolumes(partitions):
     return btrfs_subvolumes
 
 
+def parse_global_storage(gs_value):
+    """ Something in the chain is converting on and off to true and false.  This converts it back.
```

Comment:
It's not really parsing, though, is it? More like "munge". Or `bool_to_zfs_command_boolean()`